exchanges/exchange.go

```diff
@@ -458,7 +460,7 @@ func (b *Base) GetRequestFormattedPairAndAssetType(p string) (currency.Pair, ass
 			}
 		}
 	}
-	return response, "", errors.New("pair not found: " + p)
+	return response, "", ErrPairNotFound
```

Comment:
Please wrap for other calls outside of scope. 
```go
return response, "", fmt.Errorf("%s %w", p, ErrPairNotFound)
```

Reply:
> Thanks for opening this PR, I am thinking maybe in this instance we can add in a function specifically for this use case that uses Available Pairs instead of Enabled Pairs. So we don't suppress errors, the other changes can still occur. Let me know if this will cause issues on your end.

Bit on the fence about using `AvailablePairs`, nice thing about the current setup is that we spin up the GCT-wrapping service without the need for knowledge of unrelated pairs, if we want it to handle only BTC and ETH that's really only it needs to know, on startup the service tells GCT that it only cares about BTC/ETH and nothing else.